.github/workflows/ci.yml

```diff
@@ -49,26 +56,31 @@ jobs:
   
   eip-validator:
     name: EIP Validator
+
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout EIP Repository
-      uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-    
-    - name: Set Up Rust Toolchain
-      uses: actions-rs/toolchain@b2417cde72dcf67f306c0ae8e0828a81bf0b189f
-      with:
-        toolchain: stable
-    
-    - name: Install EIP Validator
-      run: cargo install eipv --version=0.4.0
-    
-    - name: Run EIP Validator
-      run: eipv EIPS/ --ignore=title_max_length,missing_discussions_to --skip=eip-20-token-standard.md
+      - name: Checkout EIP Repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          ref: "refs/pull/${{ github.event.number }}/merge"
+
+      - name: Set Up Rust Toolchain
+        uses: actions-rs/toolchain@b2417cde72dcf67f306c0ae8e0828a81bf0b189f
+        with:
+          toolchain: stable
+
+      - name: Install EIP Validator
+        run: cargo install eipv --version=0.4.0
+
+      - name: Run EIP Validator
+        run: eipv EIPS/ --ignore=title_max_length,missing_discussions_to --skip=eip-20-token-standard.md
```

Comment:
Why was this indented?  At a glance, the original indentation was also correct I think?  YAML is dumb, but I believe both of the following are correct forms for a list
```yml
key:
- item1
- item2
- item3
```
```yml
key:
  - item1
  - item2
  - item3
```

Reply:
Because the indentation was the latter for every single other job, so I matched them for consistency.